### PR TITLE
Add doc for feature automatically upgrade volumes' engine to the default engine image after upgrading Longhorn

### PR DESCRIPTION
**Pull request by PhanLe1010**
Longhorn/longhorn#2152

### CODE REVIEW

File: content/docs/1.1.1/deploy/upgrade/_index.md [hunk 1]
```diff
@@ -7,16 +7,25 @@ Here we cover how to upgrade to latest Longhorn from all previous releases.
...
+### 2. Manually Upgrade Longhorn Engine
 
 After Longhorn Manager is upgraded, Longhorn Engine also needs to be upgraded [using the Longhorn UI.](./upgrade-engine)
 
+# Note:
+Since Longhorn v1.1.1, we provide an option to help you automatically upgrade all Longhorn volumes (except for the [disaster recovery volumes](../../snapshots-and-backups/setup-disaster-recovery-volumes/)) to the new default engine version after upgrading Longhorn manager.
+This feature reduces the amount of manual work you have to do when upgrading Longhorn.
+To enable this feature, you just need to turn on the setting [Automatically upgrade volumes' engine to the default engine image after upgrading Longhorn](../../references/settings/#automatically-upgrade-volumes-engine-to-the-default-engine-image-after-upgrading-longhorn) in Longhorn UI.
+
+Longhorn doesn't automatically upgrade disaster recovery volumes to the new default engine image because it would trigger a full restoration for the disaster recovery volumes.
```
Comment (yasker) [inline content/docs/1.1.1/deploy/upgrade/_index.md:25]: Need to mention the concurrent limit of the automated engine upgrade.

Comment (PhanLe1010) [reply content/docs/1.1.1/deploy/upgrade/_index.md]: Thank you. I fixed it

**Review by shuo-wu — APPROVED** [review]

**Comment by yasker** [comment]
@PhanLe1010 there is a conflict due to the recent re-order setting reference page. Can you rebase?

**Comment by PhanLe1010** [comment]
@yasker Done

File: content/docs/1.1.1/deploy/upgrade/auto-uprade-engine.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#### 1. Concurrent Automatic Engine Upgrade Per Node Limit Setting
+
+This is a setting that controls how Longhorn automatically upgrades volumes' engines to the new default engine image after upgrading Longhorn manager.
+The value of this setting specifies the maximum number of engines per node that are allowed to upgrade to the default engine image at the same time.
+If the value is 0, Longhorn will not automatically upgrade volumes' engines to the default version.
+The bigger this value is, the faster the engine upgrade process finishes.
+
+However, giving a bigger value for this setting will consume more CPU and memory of the node during the engine upgrade process.
+We recommend capping the value for this setting at `available_cpu/0.2` where `available_cpu` is the typical amount of idle CPU on the nodes in the cluster.
```
Comment (yasker) [inline content/docs/1.1.1/deploy/upgrade/auto-uprade-engine.md:18]: I am not sure we should ask users to calculate like this.
1. `available_cpu/0.2` will be bigger than 3 for most environments.
2. The reason for choosing value 3 is more about leaving some room for error but don't overwhelm the system with too many failed upgrades.

Comment (PhanLe1010) [reply content/docs/1.1.1/deploy/upgrade/auto-uprade-engine.md]: Okay. I removed the calculation and add the explanation for the value 3

**Review by yasker — APPROVED** [review]

**Pull request merged**